### DISCUSSION TITLE
还有可用的gae ip段吗？

### DISCUSSION THREAD

**Discussion by wpyok500** [SeaHOH/GotoX#56] [一般]
还有可用的gae ip段吗？
```

**Comment by SeaHOH** [comment]
我这儿四川电信，暂时没发现有，以前的都死了，也懒得扫全网，并发不敢开大，太慢。

估计QUIC 应该有一些，但是没试过。

现在用的 SNIProxy 代替，速度还不错。

PS：普通标准代理都会被精准过滤。

**Comment by wpyok500** [comment]
@SeaHOH 能否给份sniproxy 的ip段

**Comment by SeaHOH** [comment]
没有自己扫，都是那 2500+ 里面找的，你懂的~

**Comment by wpyok500** [comment]
哦
